scripts/jenkins/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM nvidia/cuda:10.0-devel-ubuntu18.04
+RUN apt-get update && apt-get install -y \
+        bc \
+        wget \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install CMake
+RUN export CMAKE_VERSION=3.13.4 && \
```

Comment:
Cross referencing https://github.com/ECP-copa/Cabana/pull/79/files#r264784631

@junghans are you satisfied with my reply?

Reply:
Fine with me!